### PR TITLE
Adding the ttp to find group hashes.

### PR DESCRIPTION
**Pull request by masterugwee**

### CODE REVIEW

**Comment by masterugwee** [comment]
One TTP can be used to display the group hashes from /etc/gshadow in Linux system. 
The second one can be used to display the list of groups from /etc/group.

**Review by privateducky — APPROVED** [review]
Awesome. On to @khyberspache for final approval.

File: discovery/83c991b4-5ff0-4fe5-80ab-c78737ccef61.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+id:83c991b4-5ff0-4fe5-80ab-c78737ccef61
+metadata:
+  version: 1
+  authors: 
+    - Varun
+  tags: []
+name: List group password hashes
+description: |
+  The group hashes for all groups present will be displayed from /etc/gshadow file
+technique:
```
Comment (khyberspache) [inline discovery/83c991b4-5ff0-4fe5-80ab-c78737ccef61.yml:10]: missing tactic

File: discovery/8fc7de44-c830-47c9-b05b-b53b1352e874.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+id:8fc7de44-c830-47c9-b05b-b53b1352e874
+metadata:
+  version: 1
+  authors: 
+    - Varun
+  tags: []
+name: View the group file
+description: |
+  The group names of all groups present in the system will be displayed from /etc/group file
+technique:
```
Comment (khyberspache) [inline discovery/8fc7de44-c830-47c9-b05b-b53b1352e874.yml:10]: missing tactic.

**Review by khyberspache — CHANGES_REQUESTED** [review]

File: discovery/8fc7de44-c830-47c9-b05b-b53b1352e874.yml (hunk 2 above)
Comment (masterugwee) [reply discovery/8fc7de44-c830-47c9-b05b-b53b1352e874.yml:11]: Added tactic to both files.

**Comment by khyberspache** [comment]
@masterugwee The TTPs look good, but it appears that you need to rebase because there are two extra TTPs in this PR that aren't up to date with master.  You can either:
1. Open a new PR with just the two YMLs you want to submit
2. Rebase your branch to master then repush

**Review by khyberspache — CHANGES_REQUESTED** [review]
see above comment

**Comment by masterugwee** [comment]
> @masterugwee The TTPs look good, but it appears that you need to rebase because there are two extra TTPs in this PR that aren't up to date with master. You can either:
> 
>     1. Open a new PR with just the two YMLs you want to submit
> 
>     2. Rebase your branch to master then repush

I guess this solves it. Right.

**Comment by khyberspache** [comment]
Merged your other PR

**Pull request closed without merging**